christfellowshipchurch/apollos-app
# Notification Badges

### About
This PR establishes the client-side code required for clearing/setting Notification Badges on the iOS Home Screen App Icon

### Test Instructions
Run this PR locally and receive a handful of Push Notifications from a Group Message. While the app is open, you should be able to see that the Badge is setting according to the number of unread messages.

This Client Side update does not automatically increment the number from a remote notification. That code is reflected in the server-side patch referenced below

### Screenshots
[Loom Demo](https://www.loom.com/share/07ff52a702c7476f83699d6e4ac9c4fb)
![IMG_FD5FEF1F2490-1](https://user-images.githubusercontent.com/45076058/124611397-67c1eb80-de3f-11eb-820e-23e07abaf359.jpeg)

### Closes Tickets
[CFDP-1287]

### Related Tickets
[API Patch for Remote Notifications](https://github.com/christfellowshipchurch/apollos-api/pull/300)


[CFDP-1287]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1287
